worlds/musedash/__init__.py

```diff
@@ -184,7 +184,9 @@ def create_item(self, name: str) -> Item:
             return MuseDashSongItem(name, self.player, album)
 
         song = self.md_collection.song_items.get(name)
-        return MuseDashSongItem(name, self.player, song)
+        if song:
+            return MuseDashSongItem(name, self.player, song)
+        raise KeyError(name)
```

Comment:
Why not just

```py
song = self.md_collection.song_items[name]
return MuseDashSongItem(name, self.player, song)
```

That will just raise a KeyError anyway, whose message will also include the name

Reply:
Not gonna lie, just did this for this to be consistent. :P

Has been reverted.